File: java/java-tests/testData/refactoring/inlineLocal/Qualified.java
public class A {
  int f = 23;

  static class B {
    static int f = 42;
  }

  int foo() {
    int r = B.f;
    A B = this;
    return r<caret>;
  }

  public static void main(String[] args) {
    System.out.println(new A().foo());
  }
}

File: plugins/kotlin/idea/tests/testData/intentions/expandBooleanExpression/qualified.kt
// AFTER-WARNING: Name shadowed: b
// AFTER-WARNING: Parameter 'b' is never used
// AFTER-WARNING: Variable 'b' is never used
fun test(b: Boolean) {
    val b = Foo().bar()<caret>
}

class Foo {
    fun bar() = true
}